src/app/containers/Timestamp/index.jsx

```diff
@@ -29,22 +30,60 @@ const formatTimestamp = dateObj => {
   return `${dayNumeric} ${monthLong} ${fullYear}`;
 };
 
-const TimestampContainer = ({ timestamp }) => {
-  const dateObj = new Date(timestamp);
+const isTenHoursAgo = milliseconds => {
+  const now = Date.now();
+  return now - milliseconds >= 10 * 60 * 60 * 1000;
+};
+
+const timestampWithPrefixUpdated = (datetime, milliseconds) => (
+  <Timestamp datetime={datetime} prefix="Updated">
+    {milliseconds}
+  </Timestamp>
+);
+
+const hasBeenUpdated = (updated, published) => updated !== published;
+
+const updatedTimestamp = (updated, published) => {
+  if (!hasBeenUpdated(updated, published)) {
+    return null;
+  }
 
-  if (!isValidDateTime(dateObj)) {
+  if (isTenHoursAgo(updated)) {
+    return timestampWithPrefixUpdated(
+      formatDateTime(new Date(updated)),
+      formatTimestamp(new Date(updated)),
+    );
+  }
+
+  return timestampWithPrefixUpdated(
+    formatDateTime(new Date(updated)),
+    relativeTime(updated),
+  );
+};
+
+const TimestampContainer = ({ updated, published }) => {
+  if (
+    !isValidDateTime(new Date(updated)) ||
+    !isValidDateTime(new Date(published))
+  ) {
     return null;
   }
 
+  const publishDate = new Date(published);
```

Comment:
I'd avoid using `new Date()` everywhere and instead pass the timestamp itself around, and let the functions convert to datetime if they need to.